docs/proposals/ose-rebase.md

```diff
@@ -21,24 +21,173 @@ There are different types of commits that need to be carried on top of Origin ma
 
 In order to properly handle the additional commits, we need to set naming requirements
 for the messages of those commits. In particular:
-* long-term carries should start with a `[CARRY]` prefix followed by a prefix that denotes
-the type of the commit. For example, `[CARRY][BUILD] Tooling updates` denotes that this
-commit needs to be carried long-term and is specific to the building process of the repo.
-* updates to any existing [CARRY] commit should start with a `[SQUASH]` prefix followed
+
+1. long-term carries should start with a `[CARRY]` prefix followed by a prefix that denotes
+the type of the commit, eg.
+```
+[CARRY][BUILD] Tooling updates
+```
+The above example denotes that this commit needs to be carried long-term and is specific
+to the building process of the repo.
+
+2. updates to any existing `[CARRY]` commit should start with a `[SQUASH]` prefix followed
 by the type of the commit that they are supposed to squash to. For example, if the commit
 from the example above needs to be updated, a developer can simply create a commit that
-starts with `[SQUASH][BUILD]` followed by any message.
-* commits that are not going to be carried long-term but are necessary until the next rebase
-lands, should use a `[DROP]` prefix. Note that these commits are going to be dropped
-automatically by the rebase process so you should be sure when to use this prefix.
+starts with `[SQUASH][BUILD]` followed by any message, eg.
+```
+[SQUASH][BUILD] More tooling updates
+```
+You can have mutliple `[SQUASH]` commits with the same type scattered between the carries.
+All of them will be squashed in their original commit order, into their respective
+`[CARRY]` commits.
+
+3. commits that are not going to be carried long-term but are necessary until the next
+rebase lands, should use a `[DROP]` prefix, eg.
+```
+[DROP] web console bump commit
+```
+Unlike `[CARRY]` or `[SQUASH]` commits, `[DROP]` commits are not required to use types.
+Note that these commits are going to be dropped automatically by the rebase process so you
```

Comment:
> I am mentioning this below.
> 
>     OSE builds run weekly every Monday, Wednesday, and Friday with the possibility to switch into daily builds

That was confusing for me, since once you were talking about builds and once about rebases. Could you make it more clearer that the build is the actual reabase in that case, or just drop the rebase wording?